src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+app installable and usable by all users, regardless of screen size or input type.
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Starts fast, stays fast
+
+Performance plays a significant role in the success of any online experience,
+as high performing sites engage and retain users better than poorly performing
+ones. Sites should focus on optimizing for user-centric performance metrics.
+
+{% endDetailsSummary %}
+
+Performance plays a significant role in the success of any online experience,
+as high performing sites engage and retain users better than poorly performing
+ones. Sites should focus on optimizing for user-centric performance metrics.
+
+#### Why
+
+Speed is critical for actually getting users to _use_ your experience.
+In fact, as page load times go from 1 second to 10 seconds, the probability of
+a user bouncing increases by 123%. Performance doesn21ecac977bcat stop `onload`.
+Users should never wonder whether their interaction—for example, clicking a
+button—was registered or not. Scrolling and animation should feel smooth.
+Performance affects your entire experience, from how users perceive your
+application to how it actually performs.
+
+While all applications have different needs, the performance audits in
+Lighthouse are based on the
+[RAIL user-centric performance model](https://developers.google.com/web/fundamentals/performance/rail),
+and scoring high on those audits will make it more likely that your users have
+an enjoyable experience. You can also use
+[PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/)
+or the [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report/)
+to get real-world performance data for your web app.
+
+#### How
+
+Follow our [guide on fast load times](/fast/) to learn how to make your PWA
+start fast and stay fast.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Works in any  browser
+
+Users can use any browser they choose to access your web app before it's
+installed.
+
+{% endDetailsSummary%}
+
+Users can use any browser they choose to access your web app before it's
+installed.
+
+#### Why
+
+Progressive Web Apps are web apps first, and that means they need to work
+across browsers, not just in one of them.
+
+An effective way of doing this is by, in the words of Jeremy Keith in
+[Resilient Web Design](https://resilientwebdesign.com/), identifying the core
+functionality, making that functionality available using the simplest possible
+technology, and then enhancing the experience where possible. In many cases,
+this means starting with just HTML to create the core functionality, and
+enhancing the user experience with CSS and JavaScript to create a more
+engaging experience.
+
+Take form submission for example. The simplest way to implement that is an
+HTML form that `POST`s on submit. After building that, you can enhance the
```

Comment:
"`POST`s on submit" --> "submits a `POST` request"

This makes the name more technically accurate and avoids mixing character types in the same word.